source/local/local.go

```diff
@@ -53,35 +103,35 @@ func (s Source) Discover() (source.Features, error) {
 	}
 
 	for _, file := range files {
-		hook := file.Name()
-		hookFeatures, err := runHook(hook)
+		fileName := file.Name()
+		lines, err := runHook(fileName)
 		if err != nil {
-			log.Printf("ERROR: source hook '%v' failed: %v", hook, err)
+			log.Printf("ERROR: source local failed running hook '%v': %v", fileName, err)
 			continue
 		}
-		for feature, value := range hookFeatures {
-			if feature[0] == '/' {
-				// Use feature name as the label as is if it is prefixed with a slash
-				features[feature[1:]] = value
-			} else {
-				// Normally, use hook name as label prefix
-				features[hook+"-"+feature] = value
+
+		// Append features
+		for k, v := range parseFeatures(lines, fileName) {
+			if old, ok := features[k]; ok {
+				log.Printf("WARNING: overriding label '%s': value changed from '%s' to '%s'",
```

Comment:
minor thing, but you could probably be a bit more descriptive by stating that the overridden value is from some other hook:
```"WARNING: overriding label '%s' from another hook...```
or smth

Reply:
Done